Decred Wallet/Extensions/UILabel.swift

```diff
@@ -9,12 +9,29 @@
 import UIKit
 
 extension UILabel {
+    
```

Comment:
Empty lines at line 12 and 16 can be deleted.

Reply:
resolved in 58045fa